### PR TITLE
Detail: Fix updating compound property visibility or justification

### PR DESCRIPTION
**Pull request by EvanOxfeld**
- [x] @sfeng88 @rygim @jharwig 
- [x] @srfarley @joeferner
- [x] @joeybrk372
- [x] @mwizeman @dsingley

### CODE REVIEW

File: web/war/src/main/webapp/js/detail/dropdowns/propertyForm/propForm.js [hunk 1]
```diff
@@ -545,7 +546,6 @@ define([
                             metadata: this.currentMetadata
                         }, justification)
                 });
-                self.teardown();
```
Comment (srfarley) [inline web/war/src/main/webapp/js/detail/dropdowns/propertyForm/propForm.js]: Is removing `teardown()` part of the fix?

**Comment by srfarley** [comment]
The JS test `webapp/test/unit/spec/detail/itemTest.js` is failing on Travis. Does it pass on your machine?

File: web/war/src/main/webapp/js/detail/dropdowns/propertyForm/propForm.js (hunk 1 above)
Comment (EvanOxfeld) [inline web/war/src/main/webapp/js/detail/dropdowns/propertyForm/propForm.js]: Yes. This component is torn down elsewhere. Tearing down immediately after `addProperty` is triggered caused a silent failure.

**Comment by EvanOxfeld** [comment]
@srfarley The test passes locally. I'll kick off another travis build just in case.

**Comment by EvanOxfeld** [comment]
The travis build failure doesn't seem specific to make change, and I can't reproduce this locally. I'm chalking the failure up to karma/phantom. Fixing `itemTest.js`, if it breaks again, is out of scope of this pull request.

**Pull request merged**